biolink-model.yaml

```diff
@@ -1617,7 +1614,6 @@ slots:
       - translator_minimal
     domain: molecular entity
     range: molecular entity
-    inverse: activity increased by
```

Comment:
@mikebada - apologies, the PR is a little hard to review as I removed many of the bidirectional tags on a subsequence commit in this PR (leftovers to be reviewed on the predicate call :))